src/modules/proto_control/radmin.c

```diff
@@ -828,8 +828,9 @@ int main(int argc, char **argv)
 
 	char const	*raddb_dir = RADIUS_DIR;
 	char const	*dict_dir = DICTDIR;
+#ifdef USE_READLINE_HISTORY
 	char 		history_file[PATH_MAX];
```

Comment:
What about line 1290?

Reply:
take a look at 1066.

Reply:
That's irrelevant. Put `#undef USE_READLINE_HISTORY` above `int main()`, and try to recompile it.

Reply:
Well, if we are compiling against a system without libreadline. It is supposed to declare the stubs functions:

```
#ifndef USE_READLINE_HISTORY
#	define add_history(line)
#	define stifle_history(line)
#	define read_history()
#	define write_history(history_file)
#endif
```

Therefore, we shouldn't keep `char history_file[PATH_MAX];` to avoid `unused` build errors.

Reply:
Ah sorry. Macros not in uppercase :-(